### PR TITLE
new agent for asking about web resources

### PR DESCRIPTION
**Pull request by larryarnstein**
Please review my new agent

### CODE REVIEW

File: agents/tell_me_about_it/tell_me_about_it.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Func[fixie_query_embed] says: answer from the resource
+A: answer from the resource
+"""
+agent = fixieai.CodeShotAgent(BASE_PROMPT, FEW_SHOTS, conversational=True)
+
+@agent.register_func
+def load_doc(query: fixieai.agents.Message) -> fixieai.Message:
+    url = query.text
+    print(url)
+    response = requests.get(url)
```
Comment (juberti) [inline agents/tell_me_about_it/tell_me_about_it.py:47]: requests.head avoids the need to download the bytes

File: agents/tell_me_about_it/tell_me_about_it.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Ask Func[fixie_query_embed]: Give me a short summary of this resource #doc1
+Func[fixie_query_embed] says: answer from the resource
+A: answer from the resource
+"""
+agent = fixieai.CodeShotAgent(BASE_PROMPT, FEW_SHOTS, conversational=True)
+
+@agent.register_func
+def load_doc(query: fixieai.agents.Message) -> fixieai.Message:
+    url = query.text
+    print(url)
```
Comment (juberti) [inline agents/tell_me_about_it/tell_me_about_it.py:46]: remove print statements

File: agents/tell_me_about_it/tell_me_about_it.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Fixie agent example:
+    https://github.com/fixie-ai/fixie-examples
+"""
+
+import fixieai
+import requests
+
+BASE_PROMPT = """I am a simple agent that allows the user to ask questions web a resource. 
+I ask Func[load_doc] to load the resource from a url, and then always reach out to func[fixie_query_embed] to
```
Comment (juberti) [inline agents/tell_me_about_it/tell_me_about_it.py:14]:
```suggestion
I ask Func[load_doc] to load the resource from a url, and then always reach out to Func[fixie_query_embed] to
```

File: agents/tell_me_about_it/tell_me_about_it.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Q: I want to ask questions about this resource: http://some_url
+Ask Func[load_doc]: http://some_url
+Func[load_doc] says: #doc1
+A: I've finished reading the resource. Go ahead and ask me about it.
+Q: Give me a short summary of this resource?
+Ask Func[fixie_query_embed]: Give me a short summary of this resource #doc1
+Func[fixie_query_embed] says: answer from the resource
+A: answer from the resource
+
+Q: tell me about http://some_url
```
Comment (juberti) [inline agents/tell_me_about_it/tell_me_about_it.py:27]: maybe try with a https URL here

**Review by juberti — COMMENTED** [review]
Very nice! A few minor comments.

**Comment by larryarnstein** [comment]
@hessamb I've accepted Justin's suggestions

File: agents/tell_me_about_it/tell_me_about_it.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
+"""A templated Fixie agent!
```
Comment (hessamb) [inline agents/tell_me_about_it/tell_me_about_it.py:1]: Please update this docstring to describe this agent.

**Review by hessamb — COMMENTED** [review]
Nice work done here! @larryarnstein

**Comment by larryarnstein** [comment]
@hessamb I've resolved all conversations but merging is still bloked. It says 1 approving review is required

**Review by mdwelsh — APPROVED** [review]

**Review by hessamb — APPROVED** [review]

**Review by hessamb — COMMENTED** [review]
approved indeed

**Pull request merged**